parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -345,6 +345,8 @@ removeContainerImage() {
 
 cleanUpImages() {
     local targetImage=$1
+    export targetImage
```

Comment:
would export targetImage=$1 work?

Reply:
Ideally I prefer passing targetImage as a parameter to cleanupImagesRun so we don't rely on global/local vars. We can fix this later if this is too much for this time.